src/dashboard/trulens/dashboard/Leaderboard.py

```diff
@@ -221,11 +221,30 @@ def sort_selected_apps(selected_apps, records):
                 else "nan"
             ),
         )
+
+        # Filter for Snowflake credits costs
+        filtered_snowflake_credits_app_df = app_df[
+            app_df["cost_currency"] == "Snowflake credits"
+        ]
+        # Calculate the total cost for Snowflake credits
+        total_cost_snowflake_credits = (
+            filtered_snowflake_credits_app_df["total_cost"].dropna().sum()
+        )
         col3.metric(
-            "Total Cost (USD)",
-            f"${format_quantity(round(sum(cost for cost in app_df.total_cost if cost is not None), 5), precision=2)}",
+            "Total Cost (Snowflake credits)",
```

Comment:
Can we put this behind a flag so it doesn't add a column that won't be used by OSS users?

Reply:
![image](https://github.com/user-attachments/assets/fa96da77-b264-4e61-8088-1fbfdb6bde79)
![image](https://github.com/user-attachments/assets/7d9e829e-47de-4c91-9ae8-a4e94791833f)
Updated - 
I made the assumption that a single record will always use the same currency so that the width of all columns  doesn't look off